tower-fallback/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "tower-fallback"
-version = "0.2.5"
+version = "0.2.6"
```

Comment:
Did something change in `tower-fallback` or its deps since v1.0.0-alpha.8? I'm not seeing it with `git diff --stat v1.0.0-alpha.8 origin/main`

Reply:
Oh is it because it depends on zebra-test which did change?

Reply:
yup